ingest/cell_metadata.py

```diff
@@ -183,18 +194,23 @@ def validate_header_keyword(self):
         """Check metadata header row starts with NAME (case-insensitive).
         :return: boolean   True if valid, False otherwise
         """
+
+        """Check all metadata header names are unique.
+        :return: boolean   True if valid, False otherwise
+        """
+
         valid = False
-        if self.headers[0].lower() == 'NAME'.lower():
+        if self.headers[0].upper() == "NAME":
             valid = True
-            if self.headers[0] != 'NAME':
+            if self.headers[0] != "NAME":
+                # ToDO - capture warning below in error report
                 msg = (
-                    f'Warning: metadata file keyword NAME provided as '
-                    f'{self.headers[0]}'
+                    f'Warning: metadata file keyword "NAME" provided as '
```

Comment:
It might be a bit more succinct to omit `Warning: ` or `Error: ` in such initial message assignments, and prepend it later in `store_validation_issue()` based on whether the issue type is `warn` or `error`.

Reply:
This is a great suggestion that I'll put into a ticket (it might be complicated for me to try and change the warning messages while Eno is also addressing suggestions for this PR). The change would decrease future churn in tests if we were to reclassify error <-> warning (but requires updating all reference issue.json files so may be more graceful to address separate from this PR).